package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "ion-react",
+  "version": "0.0.15",
+  "description": "Ion Design System in React",
+  "scripts": {
+    "rollup": "rollup -c"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/iurynogueira/ion-react.git"
+  },
+  "author": "Brisanet-Desenvolvimento",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/iurynogueira/ion-react/issues"
+  },
+  "homepage": "https://github.com/iurynogueira/ion-react#readme",
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^24.0.1",
+    "@rollup/plugin-node-resolve": "^15.0.1",
+    "@rollup/plugin-terser": "^0.4.0",
+    "@rollup/plugin-typescript": "^11.0.0",
+    "@types/react": "^18.0.27",
+    "node-sass": "^8.0.0",
```

Comment:
A biblioteca `node-sass` tem alguns problemas em versões mais recentes do node. Recomendo a substituição pela biblioteca `sass`

Reply:
@Atzac check this possibilite

Reply:
@iurynogueira solved